lexicons/com/atproto/server/deactivateAccount.json

```diff
@@ -0,0 +1,23 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.server.deactivateAccount",
+  "defs": {
+    "main": {
+      "type": "procedure",
+      "description": "Deactivates a currently active account. Used to finalize account migration with the old host after the account has been activated on the new host.",
```

Comment:
will we open this up to be the way for folks to temporarily deactivate/reactivate accounts as an account status thing? not sure if described the way it is to set near-term or long-term expectations.

Reply:
I think yes, but we shouldn't advertise it as a feature just yet